docs/timings-monitoring.md

```diff
@@ -1,12 +1,3 @@
-!!! warning
-
-    These notes are a work-in-progress.
-    This functionality is in limited release and subject to change.
-
-## Audience
-
-This documentation is aimed at researchers who want feedback on the performance of their job requests, as well as the performance context in which their job requests happen (i.e. was the server busy or quiet at the time).
-
 ## Timings
 
 Basic timing information is available from job and job request detail pages, for example:
```

Comment:
Is it worth moving this into another part of the manual or removing the page altogether for the time-being? I wouldn't expect anyone to have linked to it and it's not currently providing the end user with much actionable information.

Reply:
yeah fair enough, will remove